src/math.rs

```diff
@@ -129,7 +130,7 @@ where
     let mut xf = x.fract();
     let mut yf = y.fract();
     let mut zf = z.fract();
-    let (mut rxf, mut ryf);
+    let (mut rxf, mut ryf); // 🦀 These, I don't know how to rename these variables. What do they mean?
```

Comment:
Much of this function is plucked from wikipedia - math notation a lot of times uses similar variable names and they don't tend to mean anything specific - they're just placeholder variable names used for intermediate calculations. I believe in this case rxf and ryf are just iterative values keeping the current scaled cosine values of the fractional portion of x and y through each noise iteration. I assume the original author used r for 'ratio' but that's just a guess.